subsys/bluetooth/host/conn.c

```diff
@@ -1983,6 +1983,12 @@ struct bt_conn *bt_conn_add_sco(const bt_addr_t *peer, int link_type)
 	}
 
 	sco_conn->sco.acl = bt_conn_lookup_addr_br(peer);
+
```

Comment:
minor: drop this empty line (the style we try to follow is that there's no empty line between the assignment to a variable and a test for the variable's value). Otherwise this looks fine.

Reply:
is this documented anywhere?

Reply:
> is this documented anywhere?

@jori-nordic no, I don't think so. Zephyr coding guidelines don't seem to care about "small" details like this, and I don't think we have subsystem-specific guidelines anywhere.

Reply:
I think Bluetooth should be the trailblazers then :p
Just so we can point PR authors to it and have less noise about style in the reviews.
It could be a simple "coding style" page under here https://docs.zephyrproject.org/latest/connectivity/bluetooth/index.html